cloakcoin-qt.pro

```diff
@@ -1,150 +1,268 @@
-QT += core gui network
-QT += widgets
+########################
+# Introduction
+########################
+#
+# This is the file that controls how the Qt GUI wallet is built
+# It is the same for all platforms: use qmake to compile it to a Makefile (that will be specific to your platform)
+# Then run make to build the wallet.
+# It is splitted into sections:
+# * QT configuration: generic configuration specific to Qt
+# * Debug variables: variables you want to switch on or off to ease debugging
+# * Where to compile: the location where the various object files will be located. By default, everything goes to the "build" directory
+# * Platform defines: make the platform visible to the code via compiler DEFINE
+# * Mandatory dependencies: see README.md for more explanations, especially about specific version and how to install
+# * Optional Dependencies: turn them on or off through qmake arguments. See README.md for more explanations
+# * Security flags: various compiler flags to improve security
+# * Other compiler flags: various compiler flags, explained
+# * Translations: build the translations
+# * Files to compile: the list of all files that have to be compiled
+# * Misc: things that don't really fit anywhere else
+
+
+########################
+# QT configuration
+########################
+
+# QT modules we use. Widgets are since QT 4, but we don't support under that anyway
+QT += core gui network widgets
 TEMPLATE = app
-TARGET = cloakcoin-qt
-VERSION = 0.7.2
-INCLUDEPATH += src src/json src/qt src/tor
-DEFINES += QT_GUI BOOST_THREAD_USE_LIB BOOST_SPIRIT_THREADSAFE BOOST_THREAD_PROVIDES_GENERIC_SHARED_MUTEX_ON_WIN __NO_SYSTEM_INCLUDES
-#DEFINES += CURL_STATICLIB
-CONFIG += no_include_pwd
-CONFIG += thread
-CONFIG += static
 
-message($$QMAKESPEC)
+# The output file name. The extension depends on the OS
+TARGET = cloakcoin-qt
 
-#PRECOMPILED_HEADER = src/stable.h
+# Current app version
+VERSION = 2.1.0
```

Comment:
this VERSION is different from the displayed version, believe it was the initial bitcoin-core version